calicoctl/commands/apply.go

```diff
@@ -39,7 +40,9 @@ Examples:
 Options:
   -h --help                 Show this screen.
   -f --filename=<FILENAME>  Filename to use to apply the resource.  If set to
-                            "-" loads from stdin.
+                            "-" loads from stdin. If filename is a directory, this command is
+                            invoked for each .json .yaml and .yml file within that directory,
+                            terminating after the first failure.
```

Comment:
Do we process sub-directories or rather recursively?

Reply:
I don't personally think this should recurse

Reply:
I was looking at kubectl's behavior and [kubectl will only recurse on an explicit `--recursive` flag](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#create). So I am okay with not recursing here. Can you update the doc/comment to make the explicit?